MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -408,3 +456,11 @@ extension RouteMapViewController: RoutePageViewControllerDelegate {
         return routeController.routeProgress.currentLegProgress.stepAfter(step)
     }
 }
+
+extension Route {
+    var polyline: MGLPolyline {
```

Comment:
I agree that this method is useful, but you don’t appear to be using it anywhere. Looks like you went with passing in a raw array of coordinates in order to more easily get a subarray of them.